app/src/main/res/layout/layout_create_file_java.xml

```diff
@@ -83,5 +83,15 @@
 
   </com.google.android.material.textfield.TextInputLayout>
 
+  <CheckBox
+        android:id="@+id/checkButton"
+        android:layout_height="wrap_content"
+        android:layout_marginEnd="16dp"
+        android:layout_marginStart="12dp"
```

Comment:
Why use `16dp` for end margin but `12dp` for start?

Reply:
Which would you like best for the layout with the new material design?